cli/provider/cmd.go

```diff
@@ -377,6 +381,35 @@ func (c CmdConfigurator) ValidateFlags(ctx context.Context, cmd *cobra.Command)
 				}
 			}
 		}
+	case "Normalise":
+		path := viper.GetString("path")
```

Comment:
You need not do this if add things in config please go through test and record command

Reply:
Refactored